lib/node_modules/@stdlib/math/base/special/normhermite-polynomial/docs/repl.txt

```diff
@@ -17,9 +17,14 @@
 
     Examples
     --------
-    > var y = {{alias}}( TODO, TODO )
-    TODO
+    > var y = {{alias}}( 1, 0.5 )
+    0.5
+    > y = {{alias}}( -1.0, 0.5)
+	NaN
+	> y = {{alias}}( 0, 0.5 );
```

Comment:
Remove the semicolon. Similar to MATLAB, our idea is that semicolons "silence" printed output. Here, we want to print the return value, so we omit the semicolon. The same applies to L26.

Note that this is a forward looking concern. We have yet to rewrite the REPL environment to support such silencing.